src/helpers/storage.js

```diff
@@ -0,0 +1,20 @@
+export function setMeta(key, value) {
+  const meta = getStorage("meta") ?? {};
+  meta[key] = value;
+  setStorage("meta", meta);
+}
+
+export function getMeta(key) {
+  const meta = getStorage("meta") ?? {};
+  return meta[key];
+}
+
+function getStorage(item) {
+  const json = window.localStorage.getItem(item);
```

Comment:
Nit: will this behave properly properly on iOS browsers where IIRC local storage has a short expiration duration? Same for `setStorage` below.

Reply:
Good point... I'll mark this and create an issue